core/src/main/resources/org/apache/spark/log4j2-defaults.properties

```diff
@@ -16,7 +16,7 @@
 #
 
 # Set everything to be logged to the console
-rootLogger.level = warn
```

Comment:
Hm, after this Pr, it causes this issue https://issues.apache.org/jira/browse/SPARK-37792. I think the issue is actually from the initial migration PR, and this change accidentally hid the problem ..

Reply:
I will dig in it later.thanks